SI/Sem.02/Pract.02/Task2.cpp

```diff
@@ -0,0 +1,33 @@
+// ConsoleApplicationPrakt2Task2.cpp : This file contains the 'main' function. Program execution begins and ends there.
+//
+
+#include <iostream>
+using namespace std;
+int main()
+{
+    char symbol;
+    cout << "Enter a symbol";
+    cin >> symbol;
+    if ('A' <= symbol && symbol<= 'Z' || 'a' <= symbol && symbol <= 'z')
+    {
+        switch (symbol)
+        {
+        case 'a': ;
```

Comment:
When you declare cases like that, you do not need semicolon (;) - it is an empty statement.